ordered_set.py

```diff
@@ -38,7 +33,10 @@ def is_iterable(obj):
     )
 
 
-class OrderedSet(MutableSet, Sequence):
+T = TypeVar('T')
+
+
+class OrderedSet(MutableSet[T], Sequence[T]):
```

Comment:
Is there no way to specify that an OrderedSet[T] is a Sequence[T] in the .pyi file? We have to import `typing` in the code itself?

If so, what is the point of .pyi files? I thought the entire goal was that .pyi files could be written separately to describe a library that already exists.

Reply:
> Is there no way to specify that an OrderedSet[T] is a Sequence[T] in the .pyi file? We have to import `typing` in the code itself?
I specified this in the `.pyi` file, but that's not enough: the `.pyi` file is not executed, but the `.py` file is, and when you run `OrderedSet[int]`, for example, python looks at the `OrderedSet` definition in the `.py` file.

> If so, what is the point of .pyi files? I thought the entire goal was that .pyi files could be written separately to describe a library that already exists.
The point of `.pyi` files is that you can describe types with convenient type annotations convention even if you use old python. I don't see the point to annotate existing library, that supports only python > 3.5, that way.